ansible/group_vars/all

```diff
@@ -1 +1,2 @@
 # Default / placeholder for variables
+docker_use_tls: verify
```

Comment:
My general thoughs on `groupvars`/`hostvars` under infrastructure were not to use them, since they'll need to be duplicated in our private configs. Could you move this into a default role variable instead, or even make it mandatory if it's not likely to break anything?